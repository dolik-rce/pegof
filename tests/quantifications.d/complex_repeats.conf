input quantifications.d/complex_repeats.peg
optimize repeats
header never
