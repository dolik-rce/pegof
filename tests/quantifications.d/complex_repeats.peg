X <-
    ("A" . [A-Z]*) ("A" . [A-Z]*)*
    / ("A"* "B" / "C"+)+ ("A"* "B" / "C"+)?
