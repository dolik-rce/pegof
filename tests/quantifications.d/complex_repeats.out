X <-
    ("A" . [A-Z]*)+
    / ("A"* "B" / "C"+)+
